common/scripts/build-uefi-apps.sh

```diff
@@ -62,6 +63,20 @@ if [ $BUILD_PLAT = SR ]; then
    BUILD_PLAT=ES
 fi
 
```

Comment:
Please refactor the changes in file to a new file common/scripts/build-uefi-apps-arm.sh

The original file common/scripts/build-uefi-apps.sh shall be unmodified.